charts/lazylibrarian/values.yaml

```diff
@@ -1,153 +1,42 @@
-  calibre:
-    enabled: false
-    image:
-      repository: linuxserver/calibre-web
-      tag: calibre
-  # Enable the FFMpeg Docker Mod. This allows using the audiobook conversion features of LazyLibrarian.
-  ffmpeg:
-    enabled: false
-    image:
-      repository: linuxserver/mods
-      tag: lazylibrarian-ffmpeg
-
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
-  annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-podAnnotations: {}
-
-podSecurityContext: {}
-  # fsGroup: 2000
+  tag: version-047f91af
 
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
-
-persistence:
-  enabled: false
-  annotations: {}
-  ## lazylibrarian data Persistent Volume Storage Class
-  ## If defined, storageClassName: <storageClass>
-  ## If set to "-", storageClassName: "", which disables dynamic provisioning
-  ## If undefined (the default) or set to null, no storageClassName spec is
-  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
-  ##   GKE, AWS & OpenStack)
-  ##
-  # storageClass: "-"
-
-  ## If you want to reuse an existing claim, you can pass the name of the PVC using
-  ## the existingClaim variable
-  ##
-  # existingClaim: lazylibrarian-config
-  accessMode: ReadWriteOnce
-  size: 1Gi
-
-  # Any extra volumes to define for the pod
-  # Volumes will be mounted to the folder specified under mountPath
-  # If no mountPath is set it will be mounted to /mnt/<name>
-  extraVolumes: []
-  #  - name: example-name
-  #    hostPath:
-  #      path: /path/on/host
-  #    type: DirectoryOrCreate
-  #    mountPath: "/mnt/test"
+strategy:
+  type: Recreate
 
 service:
-  type: ClusterIP
-  port: 5299
-  # externalTrafficPolicy: Local
-  # loadBalancerIP: ""
-  # loadBalancerSourceRanges: []
-  annotations: {}
-
-ingress:
-  enabled: false
-  annotations: {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
-  hosts:
-    - host: chart-example.local
-      paths: []
-  tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
+  port:
+    port: 5299
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+env: {}
+  # TZ: UTC
+  # PUID: 1001
+  # PGID: 1001
+  # DOCKER_MODS=linuxserver/mods:lazylibrarian-ffmpeg
+  # DOCKER_MODS=linuxserver/calibre-web:calibre
```

Comment:
I don't think you can have the same env var twice, the second one would overwrite the first. Not sure about the lsio syntax to load multiple docker mods though.

Reply:
Its a comment, just to show users where to put it. 
I don't even know if those work at all, as it isn't part of any testing anyway.

Reply:
It looks like in the old code they are just appended after eachother.

Reply:
Added a warning comment about using docker mods, to shift blame in case of layer 8 errors.